packages/react-components/index.js

```diff
@@ -44,7 +44,6 @@ module.exports = (neutrino, opts = {}) => {
       neutrino.use(react, options);
 
       // Remove the html generation plugins added by the react preset
```

Comment:
@eliperelman do you know why the html plugins are manually removed here, rather than just always setting the `html` preset option to `false` (which would stop the plugins being added in the first place)?

Reply:
I'm not sure, my guess was just not thinking about it.

Reply:
I've opened #1093 for this.